index.html

```diff
@@ -167,86 +175,132 @@ <h2>Verifiable Credential</h2>
 
     <section>
       <h2>Securing JSON-LD</h2>
+      <p>
+        The use of Verifiable Credentials often involves the representation and exchange of structured 
+        data in the form of JSON-LD. While JSON-LD provides a flexible and extensible format for 
+        describing data, it is important to note that it also provides a linkage between the data structure
+        and semantic meaning of data.
+        This section outlines how JSON-LD encoded data can be secured using either JOSE or COSE though use of
+        a VC-JWT. 
+      </p>
+      <p>
+        A benefit to this approach is that credentials can be used that conform directly to the [[VC-DATA-MODEL]]
+        without need of any mapping or transformation.
+      </p>
       <section>
         <h2>With JOSE</h2>
         <section>
-          <h2>Securing <code>application/vc+ld+json</code> with JOSE</h2>
+          <h2>Securing JSON-LD VCs with JOSE</h2>
+          <p>
+            This section details how to secure data with the type <code>application/vc+ld+json</code> 
+            with JOSE.
+          </p>
           <p>[[rfc7515]] MAY be used to secure this media type.</p>
-          <p><code>typ</code> MUST be <code>vc+ld+jwt</code></p>
-          <p><code>cty</code> MUST be <code>vc+ld+json</code></p>
-          <p>See <a data-cite="rfc7515#section-4.1.10">Common JOSE Header Parameters</a> for additional details.</p>
-          <section>
-            <h2>Credential Header</h2>
-            <aside
-              class="example"
-              title="verifiable credential header (decoded JWT header)"
-            >
-              <pre data-include="./examples/vc-2.0/templates/vc+ld+jwt/header.json" data-include-format="json"></pre>
-            </aside>
-          </section>
-          <section>
-            <h2>Credential</h2>
-            <aside
-              class="example"
-              title="verifiable credential (decoded JWT claimset)"
-            >
-              <pre data-include="./examples/vc-2.0/templates/vc+ld+jwt/claimset.json" data-include-format="json"></pre>
-            </aside>
-          </section>
+          <p>
+            When using this approach, the <code>typ</code> MUST be <code>vc+ld+jwt</code>
+          </p>
+          <p>When using this approach, the <code>cty</code> MUST be <code>vc+ld+json</code>
```

Comment:
In a future PR, I think we can make this a SHOULD, since it is redundant to the `typ` normative requirement.

I don't suggest making any change now just noting this is sorta requiring `application/vc+ld+json` twice.

Reply:
Except that it's sorta requiring `vc+ld+json` twice, following the JWT spec...